catalog.html

```diff
@@ -0,0 +1,361 @@
+        <a href="#" class="main-header__wishlist">Закладки: 0</a>
+        <a href="#" class="main-header__cart">Корзина: 10</a>
+        <a href="#" class="main-header__checkout">Оформить заказ</a>
+      </div>
+
+      <div class="main-header__mid">
+        <p class="main-header__desc">Интернет-магазин  строительных материалов  и инструментов для ремонта</p>
+        <div class="main-header__contacts">
+          <a href="tel:+78125550555" class="main-header__phone">+7 (812) 555-05-55</a>
+          <p class="main-header__address">г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</p>
+        </div>
+        <nav class="user-nav">
+          <ul class="user-nav__list">
+            <li class="user-nav__item">
+              <a href="#" class="user-nav__link">Войти</a>
+            </li>
+            <li class="user-nav__item">
+              <a href="#" class="user-nav__link">Регистрация</a>
+            </li>
+            <li class="user-nav__item">
+              <a href="#" class="user-nav__link">Равшан Джамшутович</a>
+              <ul class="user-nav__submenu-list">
+                <li class="user-nav__submenu-item">
+                  <a class="user-nav__submenu-link">Мои заказы</a>
+                </li>
+                <li class="user-nav__submenu-item">
+                  <a class="user-nav__submenu-link">Личный кабинет</a>
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </nav>
+      </div>
+
+      <nav class="main-nav">
+        <ul class="main-nav__list">
+          <li class="main-nav__item">
+            <a href="#" class="main-nav__link">Главная</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="#" class="main-nav__link">Компания</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="#" class="main-nav__link">Каталог</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="#" class="main-nav__link">Новости</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="#" class="main-nav__link">Спецпредложения</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="#" class="main-nav__link">Доставка</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="#" class="main-nav__link">Контакты</a>
+          </li>
+        </ul>
+      </nav>
+
+    </header>
+
+    <main class="catalog-page">
+
+      <nav class="breadcrumbs">
+        <ul class="breadcrumbs__list">
+          <li class="breadcrumbs__item">
+            <a href="#" class="breadcrumbs__link">На главную</a>
+          </li>
+          <li class="breadcrumbs__item">
+            <a href="#" class="breadcrumbs__link">Каталог</a>
+          </li>
+          <li class="breadcrumbs__item">
+            <a href="#" class="breadcrumbs__link">Инструмент</a>
+          </li>
+          <li class="breadcrumbs__item">
+            <a href="#" class="breadcrumbs__link">Перфораторы</a>
```

Comment:
не должно быть интерактивной и ссылаться саму на себя
```suggestion
            <a class="breadcrumbs__link">Перфораторы</a>
```